src/core/Geometry.js

```diff
@@ -1043,6 +1043,51 @@ THREE.Geometry.prototype = {
 
 		}
 
+		var bones = [];
+		var skinIndices = [];
+		var skinWeights = [];
+		var influencesPerVertex = 0;
+
+		if ( this.bones !== undefined && this.skinIndices !== undefined && this.skinWeights !== undefined ) {
```

Comment:
Ah that makes sense though
it's a bit weird that skinning related info, skinIndices/Weights and bones, are
separated into Geometry and Skeleton.

Prolly we should clear up them later.